src/wizards/tokenSwapDealWizard/components/tokenDetails/tokenDetails.ts

```diff
@@ -157,8 +153,12 @@ export class TokenDetails {
         this.token.cliffOf = undefined;
       }
     });
-    this.aureliaHelperService.createPropertyWatch(this.token, "address", () => {
+    this.aureliaHelperService.createPropertyWatch(this.token, "address", address => {
       this.token.amount = undefined;
+      this.getTokenInfo(address);
+    });
+    this.aureliaHelperService.createPropertyWatch(this.token, "logoURI", () => {
+      this.checkURL();
```

Comment:
Needed, because of Firefox issue
```html
<pinput-text
    input.trigger="getTokenInfo(token.address)"        <-- `getTokenInfo` did not used the updated value from `value.bind`
    value.bind="token.address & validate"                 
></pinput-text>
```